app/ChartHistory.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import moment from 'moment';
+import DatePicker from './DateTimePicker/DatePicker.jsx';
+import TimePicker from './DateTimePicker/TimePicker.jsx';
+import './chart-history.scss';
+
+class ChartHistory extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            date: moment().format('YYYY/MM/DD'),
+            time: '00:00',
+        };
+    }
+    onChange({ target }) {
+        if (target.name === 'date') {
+            this.setState({
+                date: target.value,
+            });
+        }
+
+        if (target.name === 'time') {
+            this.setState({
+                time: target.value,
+            });
+        }
+
+        this.props.onChange(`${this.state.date} ${this.state.time}`);
+    }
+    render() {
+        return (
+            <div className="ciq-chart-history">
+                <DatePicker
+                    placeholder="select date"
+                    name="date"
+                    format="DD MMMM YYYY"
```

Comment:
@brucebinary Should we read this `format` from the application?

Reply:
@shayanbinary did I answer you on this on slack? I don't understand your question.

Reply:
@brucebinary actually no, do date format (`DD MMMM YYYY`) comes from any setting?